packages/sfpowerscripts-cli/src/impl/release/ReleaseDefinition.ts

```diff
@@ -45,6 +45,11 @@ export default class ReleaseDefinition {
         return releaseDefinition;
     }
 
+    public static async getArtifactsFromReleaseDefinitionFile(pathToReleaseDefinition: string) {
```

Comment:
Do we really need this helper? Cant we just consume it from the release definition just returned.

I think its confusing, as its called artifacts and on quick glance, i thought it was a real artifacts :)